src/scripts/comparison_of_horizontal_and_vertical_programs/scenario_definitions.py

```diff
@@ -31,12 +31,6 @@ def _baseline(self) -> Dict:
                     # <-- Transition into Mode2 with the effective capabilities in HRH 'revealed' in Mode 1
                     "year_mode_switch": self.YEAR_OF_CHANGE_FOR_HSS,
 
-                    # Baseline scenario is with absence of HCW
-                    'year_HR_scaling_by_level_and_officer_type': self.YEAR_OF_CHANGE_FOR_HSS,
-                    'HR_scaling_by_level_and_officer_type_mode': 'with_absence',
-                    # todo <-- Do we want the first part of the run be with_abscence too...? (Although that will mean
-                    #          that there is actually greater capacity if we do the rescaling)
-
                     # Normalize the behaviour of Mode 2
                     "policy_name": "Naive",
                     "tclose_overwrite": 1,
```

Comment:
do we definitely want tclose to be so rigid?

Reply:
which bit do you mean?

Reply:
setting tclose to 1

Reply:
ah - I see. the '1' here just means "True" (it was from the time when the read-in from Excel of bools was unreliable). The actual value that does over-write tclose (in the line below)